ui/components/app/collectible-details/collectible-details.js

```diff
@@ -270,31 +274,49 @@ export default function CollectibleDetails({ collectible }) {
             >
               {t('contractAddress')}
             </Typography>
-            <Typography
-              color={COLORS.PRIMARY1}
-              variant={TYPOGRAPHY.H6}
-              overflowWrap={OVERFLOW_WRAP.BREAK_WORD}
-              boxProps={{
-                margin: 0,
-                marginBottom: 4,
-              }}
-              className="collectible-details__contract-link"
+            <Box
+              display={DISPLAY.FLEX}
+              flexDirection={FLEX_DIRECTION.ROW}
+              className="collectible-details__contract-wrapper"
             >
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={getTokenTrackerLink(
-                  address,
-                  currentNetwork,
-                  null,
-                  null,
-                  rpcPrefs,
-                )}
-                title={address}
+              <Typography
+                color={COLORS.PRIMARY1}
+                variant={TYPOGRAPHY.H6}
+                overflowWrap={OVERFLOW_WRAP.BREAK_WORD}
+                boxProps={{
+                  margin: 0,
+                  marginBottom: 4,
+                }}
+                className="collectible-details__contract-link"
               >
-                {inPopUp ? shortenAddress(address) : address}
-              </a>
-            </Typography>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={getTokenTrackerLink(
+                    address,
+                    currentNetwork,
+                    null,
+                    null,
+                    rpcPrefs,
+                  )}
+                  title={address}
+                >
+                  {inPopUp ? shortenAddress(address) : address}
+                </a>
+              </Typography>
+              <button
+                className="collectible-details__contract-copy-button"
+                onClick={() => {
+                  handleCopy(address);
+                }}
+              >
+                {copied ? (
+                  t('copiedExclamation')
+                ) : (
+                  <Copy size={15} color="#6a737d" />
```

Comment:
Guess we don't have any colors stored as javascript constants or a color object? I'll add that ability for the design system colors. I can see this being used in other places.

Reply:
Yes I've found this odd as well. We should probably have some javascript constants that are tightly coupled with the rest of the design system?

Reply:
I'll add that as a requirement to the design tokens work that I'm doing!